src/core/libraries/np_trophy/trophy_ui.h

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <string>
+#include <variant>
+#include <vector>
+
+#include "common/fixed_value.h"
+#include "common/types.h"
+#include "core/libraries/np_trophy/np_trophy.h"
+#include "imgui/imgui_layer.h"
+
+namespace Libraries::NpTrophy {
+
+struct TrophyInfo {
+    int trophyId = -1;
+    std::string trophyName;
+};
+
+class TrophyUI final : public ImGui::Layer {
+    bool first_render{false};
```

Comment:
unused var

Reply:
ah, i assumed this was needed for imgui_layer for some reason, thank you

Reply:
It's not! I used it to default select the button for gamepad navigation